src/js/components/validation.js

```diff
@@ -58,8 +58,17 @@ class Validation {
   }
 
   validateAll () {
-    return Array.prototype.map.call(this.$element.find(this.options.selector), this.validate, this)
-      .every(validation => validation)
+    return Array.prototype.map.call(
+      this.getFilteredInputs(), this.validate, this
+    ).every(validation => validation)
+  }
+
+  getFilteredInputs () {
+    let inputs = this.$element.find(this.options.selector)
+
+    return inputs.filter((index, input) => {
```

Comment:
According to [MDN filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) the arguments is inverse. But this way works. 💩